tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -241,6 +243,36 @@ var _ = Describe("Create machinepool",
 				}
 			})
 
+		It("List instance-types with region flag - [id:72174]",
+			labels.Low, labels.Runtime.OCMResources,
+			func() {
+				By("List the available instance-types with the region flag")
+				typesList := []string{"dl1.24xlarge", "g4ad.16xlarge", "c5.xlarge"}
+				region := "us-east-1"
+				accountRolePrefix := fmt.Sprintf("QEAuto-accr72174-%s", time.Now().UTC().Format("20060102"))
+				_, err := ocmResourceService.CreateAccountRole("--mode", "auto",
+					"--prefix", accountRolePrefix,
+					"--permissions-boundary", permissionsBoundaryArn,
+					"-y")
+				Expect(err).To(BeNil())
+				accountRoleList, _, err := ocmResourceService.ListAccountRole()
+				Expect(err).To(BeNil())
+				classicInstallerRoleArn := accountRoleList.InstallerRole(accountRolePrefix, false).RoleArn
+				availableMachineTypes, _, err := ocmResourceService.ListInstanceTypes(
+					"--region", region, "--role-arn", classicInstallerRoleArn)
+				Expect(err).To(BeNil())
+				var availableMachineTypesIDs []string
+				for _, it := range availableMachineTypes.InstanceTypesList {
+					availableMachineTypesIDs = append(availableMachineTypesIDs, it.ID)
+				}
+				Expect(availableMachineTypesIDs).To(ContainElements(typesList))
+
+				availableMachineTypes, output, err := ocmResourceService.ListInstanceTypes(
```

Comment:
I think it is another checkpoint, right? Please add the step description `By()` and keep a space line with the previous checkpoint.

Reply:
Added